OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -219,6 +219,7 @@ public class Aircraft : ITick, ISync, IFacing, IPositionable, IMove, IIssueOrder
 		public bool ForceLanding { get; private set; }
 		IEnumerable<CPos> landingCells = Enumerable.Empty<CPos>();
 		bool requireForceMove;
+		int initDelay;
```

Comment:
Likewise here - please call this `moveIntoWorldDelay` to make its meaning immediately clear.